.travis.yml

```diff
@@ -0,0 +1,18 @@
+
+language: c
+
+install:
+  - curl -sLo /tmp/mono.gpg http://download.mono-project.com/repo/xamarin.gpg
+  - sudo apt-key add /tmp/mono.gpg
+  - sudo sh -c "echo 'deb http://download.mono-project.com/repo/debian wheezy main' >> /etc/apt/sources.list.d/mono-xamarin.list"
+  - sudo apt-get update -qq
+  - sudo apt-get install -qq mono-devel mono-gmcs mono-dmcs mono-vbnc nuget libgdiplus=2.10-3
+
+env:
+  - TARGET=mono-2.0 MCS=gmcs
+  - TARGET=mono-4.0 MCS=dmcs
```

Comment:
see if it is possible to change these 2 lines to 
```
 - TARGET=mono-2.0 MCS="mcs -sdk:2"
 - TARGET=mono-4.0 MCS="mcs -sdk:4"
```
since the `gmsc` and `dmcs` programs are now depreciated and may be going away in the future.